ckb-integration-test/src/case/rfc0028/v2019.rs

```diff
@@ -0,0 +1,207 @@
+use super::{ERROR_IMMATURE, PASS};
+use crate::case::{Case, CaseOptions};
+use crate::{CKB2019, CKB2021};
+use ckb_testkit::util::since_from_relative_timestamp;
+use ckb_testkit::NodeOptions;
+use ckb_testkit::{BuildInstruction, Nodes};
+use ckb_types::core::Capacity;
+use ckb_types::packed::OutPoint;
+use ckb_types::{
+    core::TransactionBuilder,
+    packed::{CellInput, CellOutput},
+    prelude::*,
+};
+use std::collections::HashSet;
+
+pub struct RFC0028V2019;
+
+impl Case for RFC0028V2019 {
+    fn case_options(&self) -> CaseOptions {
+        CaseOptions {
+            make_all_nodes_connected: false,
+            make_all_nodes_synced: false,
+            make_all_nodes_connected_and_synced: false,
+            node_options: vec![
+                NodeOptions {
+                    node_name: String::from("node2019"),
```

Comment:
I suggest naming the node by its version, node-0.43 node-0.100. It's confusing because node 0.100 runs ckb2019 before activation. The same can be applied to all the other test cases.